spec/src/main/asciidoc/required-metrics.adoc

```diff
@@ -251,3 +251,58 @@ Note: This is a vendor specific attribute/operation that is not defined in java.
 |MBean| java.lang:type=OperatingSystem (com.sun.management.UnixOperatingSystemMXBean for Oracle Java, similar one exists for IBM Java: com.ibm.lang.management.ExtendedOperatingSystem)
 Note: This is a vendor specific attribute/operation that is not defined in java.lang
 |===
+
+
+=== (Optional) REST
+
+Metrics gathered from REST stats are optional and therefore may not be available in every implementation. These metrics that are created by gathering REST stats are to be portable between different implementations and as such are not valid as optional vendor metrics.
+
+The MicroProfile Metrics runtime will track metrics from RESTful resource method calls during runtime (i.e GET, POST, PUT, DELETE, OPTIONS, PATCH, HEAD). It is up to the implementation to decide how to enable the REST metrics.
+
+*(Optional) RESTRequests*
+[cols="1,4"]
+|===
+|Name| REST.request
```

Comment:
Any particular reason to turn the word `REST` into upper case? It's not quite an abbreviation, so I think we could leave it as lower case.
Not that I have a strong opinion on this though. Anyway if we decide for upper case, then the OpenMetrics output example below will need to be changed correspondingly

Reply:
The REST/JAX-RS person I was talking to wanted it capitalized since it is an acronym :b

Reply:
Ok then, I don't mind :)